VRCFaceTracking/OSC/OSCParams.cs

```diff
@@ -201,10 +190,11 @@ public override IParameter[] ResetParam(ConfigParser.Parameter[] newParams)
                 return null;
             }
 
-            protected BinaryBaseParameter(string paramName, Func<UnifiedTrackingData, float> getValueFunc) : base(paramName, getValueFunc)
+            protected BinaryBaseParameter(string paramName, Func<UnifiedTrackingData, float> getValueFunc)
             {
                 _paramName = paramName;
-                _negativeParam = new BaseParam<bool>(paramName + "Negative");
+                _getValueFunc = getValueFunc;
+                _negativeParam = new BaseParam<bool>(paramName + "Negative", data => getValueFunc.Invoke(data) < 0);
```

Comment:
Also another thing we could consider doing in another PR in the future: if there isn't a negative parameter parsed then it won't return the negative values on output (there might be some cases where an avatar creator wants to use the positive part of a combined param and exclude the negative bool to save a little space).

Reply:
This is already the case! Binary bits check to ensure the negative parameter is relevant before sending. If not and the value would be negative, they just return false
https://github.com/benaclejames/VRCFaceTracking/pull/125/files/5fa1874372286802b9c2d04cf6a0c7f6f3f9eca3..fe8709c89c71dcd946767fcb05435204076c54fe#diff-5ed98eabd76f774c20ea2dc83b3436d6ff97b8b5df8f0ea0d7b07facd30a0d27R115-R117

Reply:
Though maybe you mean allowing it to return a value regardless of whether or not the negative parameter is relevant or not (might be misunderstanding you here). Though as you said, can always be fixed in the future :)